src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,45 @@
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  gap: $day-gap;
+  width: 726px;
```

Comment:
I think you should make calculations with named variables to make it clear and adjustable

Reply:
yeah, let's do that :)

Reply:
done